ruby.c

```diff
@@ -411,6 +411,8 @@ usage(const char *name, int help, int highlight, int columns)
 
     if (highlight) sb = esc_standout;
 
+    printf("%s""Options:%s\n", sb, se);
+    printf("%s""(details and examples at https://docs.ruby-lang.org/en/master/ruby/options_md.html)%s\n", sb, se);
```

Comment:
Ideally, I think we should link to the released documentation for released versions, rather than always linking to the master branch docs since it may not apply to that version. So for example, when Ruby 3.5 comes out later this year, it should link to `https://docs.ruby-lang.org/en/3.5/ruby/options_md.html`

Reply:
So for now, would be 3.4?  How does it get updated as we go forward?

Reply:
This PR is superseded, and should be closed?  I think updates to `options.md` is out of scope here?  If agreed, I'll open a new PR for that.

Reply:
Yes, I think it's out of scope. But we should update it.